Singularity/Singularity/Screen/IWindowItem.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Singularity.Property;
+
+namespace Singularity.Screen
+{
+    interface IWindowItem : IUpdate, IDraw
+    {
+        //TODO implement the methods specific to IWindowItem
```

Comment:
Is it really just an abstraction combining IUpdate and IDraw? I don't think so - at least some coordinates and the size of the WindowItem should be getting derived from this right ... ?